frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```diff
@@ -32,6 +32,13 @@
   "Pool creation is not supported for OpenShift Container Storage's external mode.": "Pool creation is not supported for OpenShift Container Storage's external mode.",
   "A BlockPool is a logical entity providing elastic capacity to applications and workloads. Pools provide a means of supporting policies for access data resilience and storage efficiency.": "A BlockPool is a logical entity providing elastic capacity to applications and workloads. Pools provide a means of supporting policies for access data resilience and storage efficiency.",
   "BlockPool Creation Form": "BlockPool Creation Form",
+  "Compression status": "Compression status",
+  "Storage efficiency": "Storage efficiency",
+  "Compression eligibility": "Compression eligibility",
+  "Compression ratio": "Compression ratio",
+  "Compression savings": "Compression savings",
+  "<0><0><0> What does each state mean?</0></0><1><0>Starting replay:</0> Initiating image replication process.</1><2><0>Replaying:</0> Image replication is ongoing or idle between clusters.</2><3><0>Stopping replay:</0> Image replication process is shutting down.</3><4><0>Stopped:</0> Image replication process has shut down.</4><5><0>Error:</0> Image replication process stopped due to an error. </5><6><0>Unknown:</0> Unable to determine image state due to an error. Check your network connection and remote cluster RBD mirror daemon.</6></0>": "<0><0><0> What does each state mean?</0></0><1><0>Starting replay:</0> Initiating image replication process.</1><2><0>Replaying:</0> Image replication is ongoing or idle between clusters.</2><3><0>Stopping replay:</0> Image replication process is shutting down.</3><4><0>Stopped:</0> Image replication process has shut down.</4><5><0>Error:</0> Image replication process stopped due to an error. </5><6><0>Unknown:</0> Unable to determine image state due to an error. Check your network connection and remote cluster RBD mirror daemon.</6></0>",
```

Comment:
Why this change is in your commit? you better rebase once and run i18n

Reply:
Removed and ran `yarn i18n`. Rebased on top of latest PR#9423.